src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -584,10 +562,13 @@ class MediaBaseViewer extends BaseViewer {
             this.setMediaTime(start);
         }
         if (arguments.length === 0 || hasValidStart) {
-            this.mediaEl.play();
+            const autoPlayPromise = this.mediaEl.play();
```

Comment:
probably doesn't need to be named `autoPlayPromise`

Reply:
ok